src/TextField/EnhancedTextarea.js

```diff
@@ -56,9 +56,15 @@ class EnhancedTextarea extends Component {
   };
 
   state = {
-    height: this.props.rows * rowsHeight,
+    height: rowsHeight,
```

Comment:
@nathanmarks That's the only difference I see that would matter.
@hhaidar Is it ok with you to use `null`?